### DISCUSSION TITLE
The interface is not fully displayed.

### DISCUSSION THREAD

**Discussion by Lling00** [vladmandic/sdnext#415] [General]
I ran the program, but the layout of the interface shows the image area too far to the left, causing some of the parameter settings to not be fully displayed. How can I adjust this?
![image](https://user-images.githubusercontent.com/48402714/233847865-9c1cd9e9-6be7-4ac4-8f33-ce7eee4eac9d.png)

**Comment by vladmandic** [comment]
its not width thats the problem, its the content of the actual button.
its set to `↻ Reset` and that clearly does not fit in a small button. it should be just `↻`.
and if you want to see it fixed, report in the extension that draws it: <https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/>

**Reply by Lling00** [reply]
I'm trying to solve a UI layout problem, not a button functionality issue. I would like to move the image display area to the right so that the UI for the parameter section can be wider.

**Reply by vladmandic** [reply]
use a different theme or create your own by creating `user.css`.
that specific value is controlled by css variable `--left-column`
